pkg/users/users.scss

```diff
@@ -75,6 +76,14 @@
   }
 }
 
+.current-account-badge {
+  margin-left: 1.5rem;
+}
+
+.delete-account-red {
+  color: var(--pf-global--danger-color--200);
+}
```

Comment:
PatternFly recently added danger menu items (at my request), so we don't need custom CSS for that anymore. (Plus, hardcoding the color would likely mess with dark mode.)

Here's an example and the docs:
https://www.patternfly.org/v4/components/menu/#danger-menu-item

But the short summary is: In React, you'd add `isDanger`. That's it!

Reply:
Are you sure `isDanger` prop is supported for `DropDownItem`? It doesn't seem to work and I don't see it mentioned in patternfly docs.